app.js

```diff
@@ -1,46 +1,67 @@
 const bookForm = document.querySelector('#bookForm');
 const bookList = document.querySelector('#bookList');
-let books;
 
-if (localStorage.getItem('books') !== null) {
-  books = JSON.parse(localStorage.getItem('books'));
-} else {
+class BookStore {
   books = [];
-}
 
-function showBookList() {
-  bookList.innerHTML = '';
+  title = '';
+
+  author = '';
+
+  set title(title) {
+    this.title = title;
+  }
+
+  set author(author) {
+    this.author = author;
+  }
 
-  books.forEach((b, index) => {
-    bookList.innerHTML += `
-        <p>${b.title}</p>
-        <p>${b.author}</p>
-        <button type="button" onclick="removeBook(${index})">Remove</button>
-        <hr>
+  showBookList(bookList) {
+    bookList.innerHTML = '';
+
+    this.books.forEach((b, index) => {
+      bookList.innerHTML += `
+        <tr class="p-2 align-middle">
+          <td>"${b.title}" by ${b.author}</td>
+          <td><button class="btn btn-danger float float-end" type="button" onclick="remove(${index})">Remove</button></td>
+        </tr>
     `;
-  });
-}
```

Comment:
- [ ] You have submitted a project that doesn't include commits from both teammates while this activity is pair-programming 😨 Kindly ask your partner to make a commit on this pull request as suggested by the previous reviewer because I have seen that this pull request has been invalidated multiple times 🤔 Kindly ask your partner to make commit in this pull request otherwise next reviewer will invalidate it again. If you are having issues with your partner then you can reach out to StudentSuccessTeam or if you don't know how to handle the situation then you can search my name Hammas in the full-time-students channel in slack so i can guide you 😊